core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftTransportPool.java

```diff
@@ -836,7 +836,7 @@ public synchronized void setIdleTime(long time) {
   }
 
   void startCheckerThread() {
-    checkThreadFactory.get();
+    var unusedRetVal = checkThreadFactory.get();
```

Comment:
This can probably be reworked after #2554 is merged as ClientContext will now have a thread pool for close/cleanup type tasks

Reply:
For the moment assigned a return value to the method call. Added note to re-work once #2554 is merged. Will create ticket to update once #2554 is merged.